p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -4876,22 +4877,18 @@ match_kind {
 ~ End P4Example
 
 These identifiers correspond to the P4~14~ match kinds with the same
-names. The semantics of these annotations is actually irrelevant for
-describing the behavior of the P4 abstract machine; how they are used
-influences only the control-plane API and the actual implementation of
-the look-up table. From the point of view of the P4 program, a look-up
-table is an abstract finite map that is given a key and produces as a
-result either an action or a "miss" indication, as described in
-Section [#sec-mau-semantics].
```

Comment:
@mbudiu-vmw I see what this paragraph is meant to convey, but worry it might cause confusion. Given the same control plane operations, two tables with different match_kinds on their keys may behave quite differently -- e.g., ternary vs. lpm. So it's not just the implementation of the look-up table, but it's semantics too.

Since this doesn't seem like a critical point from the perspective of defining the language, maybe just cut it?